static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js

```diff
@@ -0,0 +1,121 @@
+        bonzo(this.elem).addClass(
+            window.abSwitches[`ab${this.config.test.id}`]
+                ? 'abtest-item--switched-on'
+                : 'abtest-item--switched-off'
+        );
+
+        this.getElem('description').textContent = this.config.test.description;
+
+        this.getElem('name').textContent = this.config.test.id;
+        const daysTillExpiry =
+            (Date.parse(this.config.test.expiry) - new Date()) /
+            (1000 * 60 * 60 * 24);
+        this.getElem('expiry').textContent =
+            Math.floor(daysTillExpiry).toString() +
+            (daysTillExpiry === 1 ? ' day' : ' days');
+        this.getElem('expiry').setAttribute('title', this.config.test.expiry);
+
+        this.getElem(
+            'audience'
+        ).textContent = `${this.config.test.audience * 100}%`;
+        this.getElem(
+            'audience-offset'
+        ).textContent = `${this.config.test.audienceOffset * 100}%`;
+
+        const tableauUrl = `https://tableau-datascience.gutools.co.uk/views/AutomatedMVTDashboard-MkII/MainMVTDashboard?:embed=y&id=${this.config.test.id}`;
+        this.getElem('tableau').innerHTML = `<a href="${tableauUrl}">view</a>`;
+
+        const ophanUrl = `https://dashboard.ophan.co.uk/graph/breakdown?ab=${this.config.test.id}`;
+        this.getElem('ophan').innerHTML = `<a href="${ophanUrl}">graph</a>`;
+
+        this.getElem('hypothesis').textContent =
+            this.config.test.hypothesis || '';
+
+        const participation = new Participation({
+            test: this.config.test,
+        });
+        participation.render(this.getElem('participation'));
+    }
+
+    renderChart(): void {
+        if (this.chart) {
+            new window.google.visualization.LineChart(
+                this.getElem('chart')
+            ).draw(
+                window.google.visualization.arrayToDataTable(this.chart.data),
+                {
+                    colors: this.chart.colors,
+                    curveType: 'function',
+                    chartArea: {
+                        width: '100%',
+                        height: 160,
+                        top: 0,
+                        left: 15,
+                    },
+                    legend: {
+                        position: 'in',
+                    },
+                    vAxis: {
+                        title: 'Pageviews/session',
+                        textPosition: 'in',
+                        titleTextStyle: {
+                            fontSize: 11,
+                        },
+                    },
+                    fontName: 'Helvetica',
+                }
+            );
+        }
+    }
+}
+
+ABTestReportItem.prototype.config = {
+    test: {},
+    active: true,
+};
+ABTestReportItem.prototype.templateName = 'abtest-item-template';
+ABTestReportItem.prototype.componentClass = 'abtest-item';
+ABTestReportItem.prototype.useBem = true;
```

Comment:
I've pulled these into the constructor and assigned them to 'this' rather than the prototype.